packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -626,7 +626,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         // So once we release 0.40 container-defn package we can remove this check.
         if (!this._storage && this.context.storage) {
             // Note: BlobAggregationStorage is smart enough for double-wrapping to be no-op
-            this._storage = BlobAggregationStorage.wrap(this.context.storage, this.logger);
+            this._storage = BlobAggregationStorage.wrap(this.context.storage, this.logger,
+                undefined, this.disableIsolatedChannels ? 1 : 2);
```

Comment:
Add /* allowPacking / after undefined. See other places where we use / */ after parameters.